### PR TITLE
Add Team2 code

### PR DESCRIPTION
**Pull request by GavinVasandani**

### CODE REVIEW

**Comment by arnavxkohli** [comment]
@ADimoska please enable squashing of commits

File: gameRecorder/DataVisualizer.go [hunk 1]
```diff
@@ -735,122 +769,3 @@ func generateLineItems(xAxis []int, yAxis []float64) []opts.LineData {
...
-			}
-			row = append(row, strValue)
-		}
-		if err := writer.Write(row); err != nil {
-			return err
-		}
-	}
-
-	return nil
-}
```
Comment (johanjino) [inline gameRecorder/DataVisualizer.go:856]: Is this intentional? Not sure why team2 agent code would change visualization stuff

Comment (johanjino) [reply gameRecorder/DataVisualizer.go]: Okay nvm , resolved
Cool

Comment (arnavxkohli) [reply gameRecorder/DataVisualizer.go]: Yeah sorry we removed it cuz it wasn't working before

**Comment by arnavxkohli** [comment]
Most of the stuff is within our own files, shouldn't really affect the rest of the teams. We've tried to have enough safeguards but without unit-testing we are basically just praying that everything is correct.

**Review by arnavxkohli — APPROVED** [review]

**Review by hk221 — APPROVED** [review]

**Comment by arnavxkohli** [comment]
Merged to ensure no seg faults in normal runs due to our AoA.

**Pull request merged**